modules/rosa-required-aws-service-quotas.adoc

```diff
@@ -6,7 +6,11 @@
 [id="rosa-required-aws-service-quotas_{context}"]
 = Required AWS service quotas
 
-The table below describes the AWS service quotas and levels required to create and run an {product-title} cluster.
+The table below describes the AWS service quotas and levels required to create and run one {product-title} cluster. While most default values are suitable for most workloads, you may need to request additional quota for the following cases:
+
+* ROSA clusters require at least 100 vCPUs, but the default maximum value for vCPUs assigned to Running On-Demand Standard Amazon EC2 instances is `5`. Therefore if you have not created a ROSA cluster using the same AWS account previously, you must request additional EC2 quota for `Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances`.
+
+* Some optional cluster configuration features, such as custom security groups, may require you to request additional quota. For example, because ROSA associates 1 security group with network interfaces in worker machine pools by default, and the default quota for `Security groups per network interface` is `5`, you can add up to 4 additional custom security groups for worker nodes without requesting additional quota for `Security groups per network interface`.
```

Comment:
```suggestion
* Some optional cluster configuration features, such as custom security groups, might require that you request additional quota. For example, because ROSA associates 1 security group with network interfaces in worker machine pools by default, and the default quota for `Security groups per network interface` is `5`, you can add up to 4 additional custom security groups for worker nodes without requesting additional quota for `Security groups per network interface`.
```